Src/FluentAssertions/Types/MethodInfoSelector.cs

```diff
@@ -128,6 +128,17 @@ public MethodInfoSelector ThatAreNotDecoratedWithOrInherit<TAttribute>()
             return this;
         }
 
+        /// <summary>
+        /// Select return types of the methods
+        /// </summary>
+        /// <returns></returns>
+        public TypeSelector ReturnTypes()
```

Comment:
Could this be added to `PropertyInfoSelector` as well for consistency?

Reply:
Done